bazel/external/cpp2sky.BUILD

```diff
@@ -0,0 +1,9 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
```

Comment:
This bzl can live in cpp2sky repo instead of here, or you can just bind cpp2sky to @com_github_skyapm_cpp2sky//source:cpp2sky_data_lib